module-4/backend/week_four.md

```diff
@@ -1,19 +1,55 @@
 ## Week Four - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+1. What's your favorite project management tool?
 
-Note: When you're done, submit a PR. 
+I like Waffle for its Git integration and its less busy compared to Pivotal Tracker, especially for individual or pair projects. I've recently been introduced to Jira, and would be interested in using it for its Git and CI integration.  
```

Comment:
nice - i've never used Jira, but i just started using Clubhouse and I've been enjoying it!